CONTRIBUTING.md

```diff
@@ -53,7 +53,7 @@ pip install pipenv
 Then run the following in order to install dependencies:
 
 ```bash
-pipenv install --dev
+pipenv install --dev -e .
```

Comment:
Actually, I'm not sure this does what we want. :-)

`pipenv install --dev` installs the dependencies specified in the `dev-packages` section of `Pipfile`. But with `-e .` added at the end it installs the local package and registers it in Pipfile/Pipfile.lock.

So, I think what should work is to run the `pipenv install --dev -e .` command yourself, and commit the resulting changes to Pipfile/Pipfile.lock. That way, `.` is added to dev dependencies and will be installed with the rest when running `pipenv install --dev`. :+1: Does that make sense?

Reply:
yes, it's really right solution